### PR TITLE
feat: export xudt holders support `merge_with_owner`

### PR DESCRIPTION
**Pull request by PainterPuppets**
resloved: https://github.com/Magickbase/ckb-explorer-public-issues/issues/717

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -913,8 +913,12 @@
       "csv_export": "CSV Export",
       "download_data": "Download Data",
       "holders_by_block": "Holders by Block",
+      "holders_allocation": "Holders Allocation",
```
Comment (Keith-CY) [inline src/locales/en.json:916]: I found the word in `zh.json` is `Export` but not `Allocation`

File: src/styles/index.css [hunk 2]
```diff
@@ -148,4 +148,13 @@ a:hover,
...
+
+.ant-input-number-affix-wrapper {
+  border-radius: 4px;
+  box-shadow: none;
+}
+
+.ant-input-number-affix-wrapper.ant-input-number-affix-wrapper-focused,
+.ant-input-number-affix-wrapper:hover {
+  border-color: var(--primary-color) !important;
+}
```
Comment (Keith-CY) [inline src/styles/index.css:160]:
```suggestion
}

```

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
image

It is best to check `By CKB Address` by default on the page, because if neither is checked, the interface returns data by default according to `By CKB Address`.
Or there is an additional judgment on the front end, and one method needs to be checked before the export button can be clicked.  @PainterPuppets

**Comment by FrederLu** [comment]
image

It is best to check `By CKB Address` by default on the page, because if neither is checked, the interface returns data by default according to `By CKB Address`.
Or there is an additional judgment on the front end, and one method needs to be checked before the export button can be clicked.  @PainterPuppets

**Review by FrederLu — APPROVED** [review]

**Pull request merged**